official-antistasi-community/A3-Antistasi
# Fix garrison spawning on destroyed buildings and windowless building

## What type of PR is this.
1. [X] Bug
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
Fixes a bug in UPSMon where the building placement routine will place units on destroyed/hidden buildings, causing them to fall to their deaths.

Also added a large windowless building to the UPSMon blacklist, because it fits a whole squad and AIs don't seem inclined to enter it, which stalls out attacks on outpost_37.

### Please specify which Issue this PR Resolves.
closes #1470

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
1. Destroy a tall military building in a rebel garrison. Easy way is to point binos at it and run `cursorObject setdamage 1`.
2. Make that garrison spawn in (retreat to 1km, return to 500m). Troops should no longer fall to their deaths.
